styles/styles.css

```diff
@@ -33,3 +33,193 @@ body {
   background-size: auto;
   font-family: "Nexa Light", sans-serif;
 }
+
+header {
+  height: 90px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+  transition: background-color 0.5s;
+  font-family: "Nexa Light";
+}
+
+footer {
+  height: 550px;
+  width: 100%;
+  margin-bottom: 0%;
+  align-items: center;
+  justify-content: space-between;
+  transition: background-color 0.5s;
+}
+
+
+
+
+.header-logo {
+  align-content: center;
+  padding-top: 5px;
+  transform: translate(250px);
+}
+
+.header-left, .header-right {
+  margin: 0;
+  width: 15%;
+}
+
+.header-left a {
+  text-decoration: none;
+
+}
+
+.header-right
+{
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  display: grid;
+  padding-top: 80px;
+  padding-bottom: 10px;
+  padding-left: 80px;
+  transform: translate(190px);
+}
+
+
+.header-right a {
+  text-decoration: none;
+  padding-bottom: 70px;
+}
+
+.icons-header
+{
+  display: flex;
+  padding-top: 35px;
+  padding-right: 60px;
+}
+
+.cart-header
+{
+
+  width: 45px;
+  height: 40px;
+  padding-right: 50px;
+  
+  
+}
+
+.profile-header
+{
+  align-items: right;
+  width: 40px;
+  height: 40px;
+  padding-left: px;
+  padding-bottom: 40px;
+
+}
+
+a:active {
```

Comment:
Please use a more specific selector here, otherwise you're targeting all anchor elements across the website!